recipes/pcapplusplus/cmake/conanfile.py

```diff
@@ -0,0 +1,60 @@
+import os
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake
+from conan.tools.files import copy, get, rmdir
+from conan.errors import ConanInvalidConfiguration
+
+
+required_conan_version = ">=1.57.0"
+
+class PcapplusplusConan(ConanFile):
+    name = "pcapplusplus"
+    package_type = "static-library"
+    license = "Unlicense"
+    description = "PcapPlusPlus is a multiplatform C++ library for capturing, parsing and crafting of network packets"
+    topics = ("pcap", "network", "security", "packet")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/seladb/PcapPlusPlus"
+    settings = "os", "arch", "build_type", "compiler"
+    options = {
+        "immediate_mode": [True, False],
+    }
+    default_options = {
+        "immediate_mode": False,
+    }
```

Comment:
```suggestion
    }

    def config_options(self):
        if self.settings.os == "Windows":
            del self.options.fPIC

```
Manage fPIC in case building on Windows.

Reply:
See this comment: https://github.com/conan-io/conan-center-index/pull/20296#discussion_r1356052062

Reply:
Updated here: https://github.com/conan-io/conan-center-index/pull/20296/commits/667883dd2315215e75910859d6ed6024f8ed63e5